websites/A/Audible/metadata.json

```
@@ -5,13 +5,23 @@
     "name": "Tofuine",
     "id": "734631480541708291"
   },
+  "contributors": [
+    {
+      "name": "Tofuine",
+      "id": "734631480541708291"
+    },
+    {
+      "name": "DiaDuck",
+      "id": "791491962620346420"
+    }
```

```suggestion
    {
      "name": "DiaDuck",
      "id": "791491962620346420"
    }
```
Remove the author from contributors list